zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedOrErroredRuntimeChecker.scala

```diff
@@ -45,13 +45,37 @@ object DeletedOrErroredRuntimeChecker {
           res <- runtimeOpt match {
             case None =>
               (if (isDryRun) F.unit
-               else
-                 dbReader.markRuntimeDeleted(runtime.id)).as(Some(runtime))
-            case Some(instance) =>
-              if (instance.getStatus.getState == com.google.cloud.dataproc.v1.ClusterStatus.State.ERROR) {
+               else {
+                 for {
+                   _ <- dbReader.markRuntimeDeleted(runtime.id)
+                   _ <- dbReader.insertClusterError(
+                     runtime.id,
+                     None,
+                     s"""
+                        |Runtime(${runtime.runtimeName}) was deleted from Google. Is billing still enabled for ${runtime.googleProject.value}?
```

Comment:
Is the second sentence because we've usually seen this situation happen when billing was disabled? It wasn't immediately obvious to me.

Reply:
yea. that's my best guess..jsut to give user some idea what could be the cause....happy to add other causes if you can think of any